contracts/JBController/1.sol

```diff
@@ -695,6 +750,54 @@ contract JBController is IJBController, IJBMigratable, JBOperatable, ERC165 {
     );
   }
 
+  /**
+    @notice
+    Mint new NFT to the beneficiary account if configured as such by the funding cycle data source
+
+    @dev
+    Only a project's owner, a designated operator, or one of its terminals can mint its tokens.
+
+    @param _projectId The ID of the project to which the tokens being minted belong.
+    @param _beneficiary The account that the tokens are being minted for.
+    @param _memo A memo to pass along to the emitted event.
+
+    @return tokenId The NFT id of the newly minted token
+  */
+  function mintTokens721Of(
+    uint256 _projectId,
+    address _beneficiary,
+    string calldata _memo
+  )
+    external
+    virtual
+    override
+    requirePermissionAllowingOverride(
```

Comment:
Suggest moving this (and not use the modifier) inside the function body, to avoid calling fundingCycleStore twice for the same data

Reply:
cool, thanks

Reply:
can be changed, but the new functions were modeled on existing ones as not to rock the boat too much

Reply:
> can be changed, but the new functions were modeled on existing ones as not to rock the boat too much

https://github.com/jbx-protocol/juice-contracts-v2/blob/fc7090647d19ca3dee5e648e55ea16284973d8b5/contracts/JBController/2.sol#L645